lms/djangoapps/api_manager/users/tests.py

```diff
@@ -1439,8 +1438,7 @@ def test_user_courses_grades_list_get(self):
         grading_policy = response.data['grading_policy']
         self.assertGreater(len(grading_policy['GRADER']), 0)
         self.assertIsNotNone(grading_policy['GRADE_CUTOFFS'])
-
-        self.assertEqual(response.data['current_grade'], 0.73)
+        self.assertEqual(response.data['current_grade'], 0.7)
```

Comment:
Equality checking for floaring point numbers is brittle.  Is this guaranteed to be exactly 0.7, or can this be the result of a computation?  In the latter case, I'd suggest using `self.assertAlmostEqual()` instead.